vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -421,7 +421,7 @@ func GetAvailableResources(clientBuilder controller.ControllerClientBuilder) (ma
 // CreateControllerContext creates a context struct containing references to resources needed by the
 // controllers such as the cloud provider and clientBuilder. rootClientBuilder is only used for
 // the shared-informers client and token controller.
-func CreateControllerContext(s *options.CMServer, rootClientBuilder, clientBuilder controller.ControllerClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
+func createControllerContext(s *options.CMServer, rootClientBuilder, clientBuilder controller.ControllerClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
```

Comment:
I assume this carry replaces https://github.com/openshift/origin/commit/ce08d2fdb2990ff8b1b7fcad79236f87c0a621a6 ?